index.js

```diff
@@ -1,17 +1,49 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort((a, b) => a - b);
+    this.length = this.length;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos >= this.items.length) {
+      throw new Error("EmptySortedList");
+    }
 
-  max() {}
+    return this.items.indexOf(pos);
+  }
 
-  min() {}
+  /*La nueva sintaxis extendida es una manera más corta de escribir la solución con apply para obtener el máximo de un arreglo: Sin embargo, tanto la sintaxis extendida (...) como apply fallarán o retornarán un resultado incorrecto si el arreglo tiene demasiados elementos, porque intentan pasar el arreglo de elementos como parámetros de función. Ver apply y funciones incorporadas para más detalles. La solución con reduce no tiene este problema.*/
+  max() {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    }
+    return Math.max(...this.items);
```

Comment:
Esto esta perfecto!
Pero te voy a hacer 3 preguntas:
1º ¿Que son los tres puntos que pones delante del this.items?
2º ¿Porque son necesarios esos puntos para la funcion Math.max()
3º Has dicho que se podria hacer con un reduce, ¿sabrias explicarlo y decirme como?

Siempre siempre siempre que vayas a hacer un problema, preguntate como lo haria de la version mas sencilla y luego lo vamos cambiando o "refactorizamos" asi es como se llama vale? :D